assets/constants.js

```diff
@@ -532,6 +532,7 @@ var constants = {
   "NUM_QUESTIONS_PER_PAGE": 10,
 
   "NEW_STATE_TEMPLATE": {
+      "ask_learners_for_response": false,
```

Comment:
Can we call this solicit_explanations or solicit_answer_details instead? "Response" is too generic and could just as easily connote the student's answer to a question.

Reply:
I think ```solicit_answer_details``` seems good.

Reply:
Done